pages/resources/list.vue

```diff
@@ -73,4 +73,4 @@ export default {
 .resources-header {
   font-weight: 700;
 }
-</style>
\ No newline at end of file
+</style>
```

Comment:
Just FYI - there is an ESLint rule you can enable to enforce this: https://eslint.org/docs/rules/eol-last

Reply:
Thanks. Getting eslint in general working on this site is a little bit of a project ;)